README.md

```diff
@@ -127,7 +126,7 @@ import { WordsToNumber } from "@persian-tools/persian-tools";
 
 WordsToNumber.convert("یگصد و بنجاه هزار", { fuzzy: true }) // "150000"  
 WordsToNumber.convert("دویشت ر بیشت هزار", { fuzzy: true }) // "220000"  
-WordsToNumber.convert("منقی ضد", { fuzzy: true }) // "-100"  
+WordsToNumber.convert("منقی صد", { fuzzy: true }) // "-100"  
```

Comment:
this misspell is intentional to show how the fuzzy option works

Reply:
sorry, my bad.
I'll fix it today, but there were test errors in the project.
why the master branch must have the test errors? 
@ali-master  @mediv0

Reply:
@mediv0 Do you think one mistake is enough to show how fuzzy options works? Or I Must revert it?

Reply:
> @mediv0 Do you think one mistake is enough to show how fuzzy options works? Or I Must revert it?

maybe one is enough

Reply:
> Sorry, My bad.
> I'll fix it today, but there were test errors in the project.
> Why the master branch must have the test errors?
> @ali-master @mediv0

tests are failing because of previous commits, not yours
we can merge this after @ali-master approves your changes

Reply:
Please revert this line's changes.